application/common/models/Mfa.php

```diff
@@ -25,6 +25,8 @@ class Mfa extends MfaBase
     const EVENT_TYPE_VERIFY = 'verify_mfa';
     const EVENT_TYPE_DELETE = 'delete_mfa';
 
+    const DEFAULT_WEBAUTHN_LABEL = 'My webauthn key';
```

Comment:
I think sticking with "Security Key" as the default label might be better, as most users wont' have any idea what "webauthn" means.